fre/app/freapp.py

```diff
@@ -7,70 +7,118 @@
+                     "read/writes",
+              required = True)
+@click.option("-r", "--remap_dir",
+              type = str,
+              help = "`fregridRemapDir` / `remap_dir` (env var) directory containing remap file" + \
+                     " for regridding",
+              required = True)
+@click.option("-s", "--source",
+              type = str,
+              help = "`source` / `source` (env var) source name for input target file name " + \
+                     "within input directory to target for regridding. the value for `source` " + \
+                     "must be present in at least one component's configuration fields",
+              required = True)
+@click.option("-g", "--grid_spec",
+              type = str,
+              help = "`gridSpec` / `grid_spec` (env var) file containing mosaic for regridding",
+              required = True)
+@click.option("-I", "--def_xy_interp",
+              type = str,
+              help = "`defaultxyInterp` / `def_xy_interp` (env var) default lat/lon resolution " + \
+                     "for output regridding. (change me? TODO)",
+              required = True)
+@click.pass_context
+def regrid(context,
+              input_dir, output_dir, begin, tmp_dir,
+              remap_dir, source, grid_spec, def_xy_interp ):
+    ''' regrid target netcdf file '''
+    context.forward(_regrid_xy)
 
 @app_cli.command()
 @click.option("-i", "--infile",
-              type=str,
-              help="Input NetCDF file containing pressure-level output to be masked",
-              required=True)
+              type = str,
+              help = "Input NetCDF file containing pressure-level output to be masked",
+              required = True)
 @click.option("-o", "--outfile",
-              type=str,
-              help="Output file",
-              required=True)
+              type = str,
+              help = "Output file",
+              required = True)
 @click.option("-p", "--psfile",
-              help="Input NetCDF file containing surface pressure (ps)",
-              required=True)
+              help = "Input NetCDF file containing surface pressure (ps)",
+              required = True)
 @click.pass_context
 def mask_atmos_plevel(context, infile, outfile, psfile):
-    # pylint: disable=unused-argument
+    # pylint: disable = unused-argument
     """Mask out pressure level diagnostic output below land surface"""
     context.forward(mask_atmos_plevel_subtool)
 
 
 @app_cli.command()
 @click.option("-i", "--inf",
-              type=str,
-              required=True,
-              help="Input file name")
+              type = str,
+              required = True,
+              help = "Input file name")
 @click.option("-o", "--outf",
-              type=str,
-              required=True,
-              help="Output file name")
+              type = str,
+              required = True,
+              help = "Output file name")
 @click.option("-p", "--pkg",
```

Comment:
Similar comment to before, `-p` could be `platform`, `-p` could be `psfile`, etc. 

To make more clear 
`-p` --> `-pkg`
`--pkg` --> `--tavg-pkg`